blaze-core/src/test/scala/org/http4s/blazecore/util/DumpingWriter.scala

```diff
@@ -30,7 +30,7 @@ import scala.concurrent.Future
 object DumpingWriter {
   def dump(p: EntityBody[IO]): IO[Array[Byte]] = {
     val w = new DumpingWriter()
-    for (_ <- w.writeEntityBody(p)) yield (w.toArray)
+    for (_ <- w.writeEntityBody(Entity(p))) yield (w.toArray)
```

Comment:
Is there any reason that `dump` wouldn't just take an `Enttiy`?

Reply:
Good idea. I wasn't thinking about refactoring much because this is a test thing, but let's do it.